### PR TITLE
Feat/st 603

### PR DESCRIPTION
**Pull request by muradalamv1**
### Change description ###

Changed the status of manage due dates

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/sptribs/caseworker/event/CaseWorkerEditDraftOrder.java [hunk 1]
```diff
@@ -18,23 +29,50 @@
...
+
+        return AboutToStartOrSubmitResponse.<CaseData, State>builder()
+            .build();
+
+    }
+
+    public SubmittedCallbackResponse draftCreated(CaseDetails<CaseData, State> details,
+                                                  CaseDetails<CaseData, State> beforeDetails) {
+        return SubmittedCallbackResponse.builder()
+            .confirmationHeader("When you have finished drafting this order,you can send it to parties in this case.")
```
Comment (paul-pearson) [inline src/main/java/uk/gov/hmcts/sptribs/caseworker/event/CaseWorkerEditDraftOrder.java:75]: Needs a space character after the comma.

File: src/main/java/uk/gov/hmcts/sptribs/common/event/page/EditDraftOrder.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+                + "<h3>IN THE MATTER OF THE NATIONAL HEALTH SERVICES (PERFORMERS LISTS)(ENGLAND) REGULATIONS 2013</h2>\n\n"
+                + "&lt; &lt; CaseNumber &gt; &gt; \n"
+                + "\nBETWEEN\n"
+                + "\n&lt; &lt; SubjectName &gt; &gt; \n"
+                + "\nApplicant\n"
+                + "\n<RepresentativeName>"
+                + "\nRespondent<hr>"
+                + "\n<h3>Main content</h3>\n\n ")
+            .optional(DraftOrderCIC::getMainContentForGeneralDirections)
+            .label("footer", "<h2>Footer</h2>\n First-tier Tribunal (Health,Education and Social Care)\n\n"
```
Comment (paul-pearson) [inline src/main/java/uk/gov/hmcts/sptribs/common/event/page/EditDraftOrder.java:38]: Space after comma

File: src/main/java/uk/gov/hmcts/sptribs/common/event/page/EditDraftOrder.java [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+        return AboutToStartOrSubmitResponse.<CaseData, State>builder()
+            .build();
+
+    }
+
+    public SubmittedCallbackResponse draftCreated(CaseDetails<CaseData, State> details,
+                                                  CaseDetails<CaseData, State> beforeDetails) {
+        return SubmittedCallbackResponse.builder()
+            .confirmationHeader("When you have finished drafting this order,you can send it to parties in this case.")
```
Comment (paul-pearson) [inline src/main/java/uk/gov/hmcts/sptribs/common/event/page/EditDraftOrder.java:59]: Space after comma

**Review by paul-pearson — CHANGES_REQUESTED** [review]

File: src/main/java/uk/gov/hmcts/sptribs/common/event/page/EditDraftOrder.java [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+    public void addTo(PageBuilder pageBuilder) {
+        pageBuilder
+            .page("editDraftOrder")
+            .pageLabel("Edit order")
+            .label("editableDraft", "Draft to be edited")
+            .complex(CaseData::getDraftOrderCIC)
+            .mandatory(DraftOrderCIC::getOrderTemplate, "")
+            .label("edit", "<hr>" + "\n<h3>Header</h3>" + "\n<h4>First tier tribunal Health lists</h4>\n\n"
+                + "<h3>IN THE MATTER OF THE NATIONAL HEALTH SERVICES (PERFORMERS LISTS)(ENGLAND) REGULATIONS 2013</h2>\n\n"
+                + "&lt; &lt; CaseNumber &gt; &gt; \n"
```
Comment (osheaf01) [inline src/main/java/uk/gov/hmcts/sptribs/common/event/page/EditDraftOrder.java:30]: Opening tag <h3>, closing tag </h2>?

**Review by osheaf01 — CHANGES_REQUESTED** [review]
Couple of minor HTML markup changes?

File: src/main/java/uk/gov/hmcts/sptribs/common/event/page/EditDraftOrder.java [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+                + "\nBETWEEN\n"
+                + "\n&lt; &lt; SubjectName &gt; &gt; \n"
+                + "\nApplicant\n"
+                + "\n<RepresentativeName>"
+                + "\nRespondent<hr>"
+                + "\n<h3>Main content</h3>\n\n ")
```
[the remark this answers is not in the document]
Comment (muradalamv1) [reply src/main/java/uk/gov/hmcts/sptribs/common/event/page/EditDraftOrder.java:36]: No this is just to draw a line.

File: src/main/java/uk/gov/hmcts/sptribs/common/event/page/EditDraftOrder.java (hunk 3 above)
Comment (muradalamv1) [reply src/main/java/uk/gov/hmcts/sptribs/common/event/page/EditDraftOrder.java:59]: is it in line 56 or line 59?

**Review by sjamihmcts — DISMISSED** [review]

**Review by geciciserkan — APPROVED** [review]

**Review by sjamihmcts — DISMISSED** [review]

**Review by ElifDenizEsea — DISMISSED** [review]

**Review by osheaf01 — DISMISSED** [review]
LGTM

File: src/main/java/uk/gov/hmcts/sptribs/common/event/page/EditDraftOrder.java (hunk 3 above)
Comment (muradalamv1) [reply src/main/java/uk/gov/hmcts/sptribs/common/event/page/EditDraftOrder.java]: This class is a part of another ticket so it has been removed

**Review by jaberelfasi — DISMISSED** [review]

**Review by ElifDenizEsea — APPROVED** [review]

**Review by paul-pearson — APPROVED** [review]

**Review by ElifDenizEsea — APPROVED** [review]

**Review by sjamihmcts — APPROVED** [review]

**Review by paul-pearson — DISMISSED** [review]

**Review by sjamihmcts — APPROVED** [review]

**Review by ElifDenizEsea — APPROVED** [review]

**Pull request merged**